content/content.md

```diff
@@ -390,58 +385,76 @@ As we add more and more of these images, we end up with something that becomes c
-- How do you computationally do a Fourier transform?
-- How do you do a Fourier transform of a whole song? (Rather than just a single note.)
+簡單來說，做傅立葉轉換能夠得到頻率，而做小波轉換，能夠得到時間和頻率。
 
-## Further 'reading'
+目前離散小波轉換（DWT）常被用於訊號編碼，像是工程和計算機科學，而連續小波轉換（CWT）常被用於訊號分析，即科學研究類。尤其在生醫領域中，心電圖 (Electrocardiography) 與腦波圖 (Electroencephalography) 是兩項常見的生醫應用。
 
-To learn more, some really good resources you can check out are:
+## 延伸閱讀
+
+要瞭解到更多的訊息，這裡有一些非常好用的資源可以參考(作者推薦的資源是英文版的)：
 
 [An Interactive Guide To The Fourier Transform](https://betterexplained.com/articles/an-interactive-guide-to-the-fourier-transform/)
-A great article that digs more into the mathematics of what happens.
+從數學角度更加深刻的介紹傅立葉轉換。
 
 [But what is the Fourier Transform? A visual introduction.](https://www.youtube.com/watch?v=spUNpyF58BY)
-A great Youtube video by 3Blue1Brown, also explaining the maths of Fourier transforms from an audio perspective.
+Blue1Brown 制作的 YouTube 影片，從音頻的角度解釋傅立葉轉換的數學原理。
 
 [A Tale of Math & Art: Creating the Fourier Series Harmonic Circles Visualization](https://alex.miller.im/posts/fourier-series-spinning-circles-visualization/)
-Another article explaining how you can use epicycles to draw a path, explained from a linear algebra perspective.
+從線性代數的角度解釋如何用周轉園來畫出形狀
 
 [Fourier transform (Wikipedia)](https://en.wikipedia.org/wiki/Fourier_transform)
-And of course, the Wikipedia article is pretty good too.
+當然，為基本科的解釋也很不錯的！
+
+以下是譯者為您推薦的一些中文參考資料：
+
+[傅立葉轉換 (維基百科 中文版)](https://zh.wikipedia.org/wiki/%E5%82%85%E9%87%8C%E5%8F%B6%E5%8F%98%E6%8D%A2)
 
-## The author
+[圖解傅立葉轉換](https://hackmd.io/@sysprog/fourier-transform)
+
+## 作者
 
 <canvas id="its-meee" class="sketch" width=500 height=500></canvas>
 
-I'm Jez! Full time I work at a [search company](https://www.google.com/) in the Bay Area, and in my spare time I like making games and interactive code things like this!
+我叫Jez! 我全職在美國海灣地區的[搜尋引擎公司](https://www.google.com/) 工作，在業餘時間我喜歡製作遊戲和互相交流程式碼。
+
+這個網頁是開源的，您可以在[GitHub](https://github.com/Jezzamonn/fourier)上查看程式碼! 如果您有任何回饋意見或想提出任何問題，請隨時透過email<span id="email-text">fourier [at] jezzamon [dot] com</span>發送電子郵件給我，或在tweetr上[Twitter](https://twitter.com/jezzamonn)發一個推文給我。
+
+如果您想了解我的更多作品，請查看我的[主頁](/)，如果您想了解我下一步的工作，可以關注我的Twitter[@jezzamonn](https://twitter.com/jezzamonn)!
 
-This webpage is open-source, you can check out the code on [GitHub](https://github.com/Jezzamonn/fourier)! If you have any feedback or want to ask any questions, feel free to email me at <span id="email-text">fourier [at] jezzamon [dot] com</span>, or shoot me a tweet on [Twitter](https://twitter.com/jezzamonn).
+## 譯者
 
-If you want to see more of my work, check out my [homepage](/), and if you want to see what I'm making next, you can follow my Twitter account, [@jezzamonn](https://twitter.com/jezzamonn)!
+您好，我是培鈞！目前我是一名研究生，就讀於正修科技大學 / 電機工程所。如果有相關的知識或技術可以分享，歡迎透過email<span id="email-text">k40005238 [at] gcloud [dot] csu [dot] edu [dot] tw</span>發送電子郵件給我。
```

Comment:
For my email address, I put some anti-spam logic here so that the `<span id="email-text">fourier [at] jezzamon [dot] com</span>` section gets replaced with the actual email when a real person views the page. Would you me like to do that for your email too?

Reply:
Sure! That will be nice.

Reply:
1.
I have just edited the name to content-zh-TW.md, please check again!

2.
The title: '互動式傅立葉轉換介紹'
Translated by 馬培鈞
I added a href link at the button where I introduce myself. If it is possible, you can help me link my name to the website.
https://etc.csu.edu.tw